docs/mllib-feature-extraction.md

```diff
@@ -375,3 +375,28 @@ data2 = labels.zip(normalizer2.transform(features))
 {% endhighlight %}
 </div>
 </div>
+
+## Feature selection
+Feature selection allows selecting relevant features for use in model construction leaving out the redundant ones. The number of features to select can be determined using the validation set. Feature selection is usually applied on sparse data, for example in text classification. Feature selection reduces the size of the vector space and, in turn, the complexity of any subsequent operation with vectors. 
```

Comment:
Would you mind adding a link to Wikipedia? [http://en.wikipedia.org/wiki/Feature_selection]